index.js

```diff
@@ -28,6 +29,8 @@ app.get('/register', (req, res) => res.render('pages/register'));
 app.get('/login', (req, res) => res.render('pages/login'));
 app.get('/main', (req, res) => res.render('pages/levels/main'));
 app.get('/bb-test', (req, res) => res.render('pages/levels/bb-test'));
+// generate a random ascii face
+app.get('/cool', (req, res) => res.send(cool()));
```

Comment:
The requirements doc says `You must also incorporate the use of an API`. I'm assuming this _has_ to be a remote api. Could you setup a new nodejs project that only has the face api that we can call remotely, rather than being hosted on our same nodejs server? Should be as super easy since you're just c/p'ing code from main.js

Reply:
We can sort of 'pretend' its some api that someone else has hosted somewhere but just host it ourself.

Reply:
Sure, I can do that. Alternatively I could use an existing web service that does exactly that: https://cool-face-service.herokuapp.com/
I'll make sure to do that later today.

Reply:
Using that service works perfect :)